Lesson2/src/Lesson2.java

```diff
@@ -0,0 +1,58 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Lesson2 {
+
+
+    public static void main(String[] args) throws IOException {
+        System.out.println(sum(10, 20));
+        System.out.println(sum(Integer.MAX_VALUE, Integer.MAX_VALUE));
+        System.out.println(max(50, 49));
+        System.out.println(average(new int[]{1, 2, 3, 4, 5}));
+        System.out.println(max(new int[]{1, 2, 3, 4, 5, 100, 99}));
+        System.out.println(calculateHypotenuse());
+    }
+
+    public static int sum(int a, int b) {
+        return a + b;
+    }
+
+    public static int max(int a, int b) {
+        if (a > b) {
+            return a;
+        } else {
+            return b;
+        }
+    }
+
+    public static double average(int[] array) {
+        int sum = 0;
+        double result = 0;
+        for (int i = 0; i < array.length; i++) {
+            sum += array[i];
+            result = sum / array.length;
```

Comment:
sum / array.length will work as ints division (if sum = 10 and array.length = 3. sum, array.length will be equal to 3, not 3.(3). Also you can call   result = sum / array.length; after "for" loop

Reply:
to make sum / array.length result be double you need either to cast one of the operands to double or changesum variable type to double